src/source/PeerConnection/Rtp.c

```diff
@@ -244,6 +296,27 @@ STATUS writeFrame(PRtcRtpTransceiver pRtcRtpTransceiver, PFrame pFrame)
     if (locked) {
         MUTEX_UNLOCK(pKvsPeerConnection->pSrtpSessionLock);
     }
+    MUTEX_LOCK(pKvsRtpTransceiver->sender.statsLock);
+    pKvsRtpTransceiver->sender.outboundRtpStreamStats.totalEncodedBytesTarget += pFrame->size;
+    pKvsRtpTransceiver->sender.outboundRtpStreamStats.framesEncoded += frames;
+    pKvsRtpTransceiver->sender.outboundRtpStreamStats.keyFramesEncoded += keyframes;
+    if (fps > 0.0)
```

Comment:
Can we add braces around single line if blocks? I think clang formatting will break with this.

Reply:
+1

Reply:
i will keep forgetting it, add a linter to CI build to enforce it